mono_repo.yaml

```
@@ -8,7 +8,7 @@ github:
     - name: "Notify failure"
       runs-on: ubuntu-latest
       # Run only if other jobs have failed and this is a push or scheduled build.
-      if: (github.event_name == 'push' || github.event_name == 'schedule') && failure()
+      if: (github.event_name == 'push' || github.event_name == 'schedule') && github.ref_name == 'master' && failure()
```

```suggestion
      if: (github.event_name == 'push' || github.event_name == 'schedule') && github.ref == format('refs/heads/{0}', github.event.repository.default_branch) && failure()
```

https://stackoverflow.com/a/69995861/374798

Can we see if this works? It would resolve your concerns about missing this if we change the default branch name.

---

Any ideas @kevmoo ?

---

🤷 sorry